stdlib/public/Concurrency/Actor.cpp

```diff
@@ -304,6 +327,16 @@ static bool swift_task_isCurrentExecutorImpl(ExecutorRef executor) {
   return executor.isMainExecutor() && isExecutingOnMainThread();
 }
 
+SWIFT_CC(swift)
+static ExecutorRef swift_task_getCurrentSerialExecutorImpl() {
+  if (auto currentTracking = ExecutorTrackingInfo::current()) {
+    return currentTracking->getActiveExecutor(); // FIXME; special case main and default
+  }
+
+  // FIXME return "none"
+  return ExecutorRef::generic();
+}
```

Comment:
TODO: Thinking to get rid of this function entirely, replace with a `checkCurrentExpected(ExecutorRef) -> ExecutorRef?` that returns the current if it DID NOT match the expected one.